### ISSUE TITLE
Crash While Logging In While In A Pocket Dimension Client To Server

### ISSUE THREAD

**Issue by DPxKnightxZero** [CannibalVox/DimDoors#10]
Crash Log - https://mega.co.nz/#!Ak1ggD6Z!Ux6uMdzHUxO3G0tmGO2aOAckXZoX-NTRkrDpQZ6Xf0U

**Comment by DPxKnightxZero** [comment]
One second uploaded wrong crash ... https://mega.co.nz/#!18FDQbrK!ofVYhEDCf7lriJX0pwwxI0WrHcnGzjlklvbc9Bh1fQk

**Comment by CannibalVox** [comment]
Can you explain what's going on in this crash?  Did someone change servers using something like bungeecord?

**Comment by DPxKnightxZero** [comment]
I was using forge essentials.

**Comment by DPxKnightxZero** [comment]
Forge Beta 5

**Comment by CannibalVox** [comment]
Okay, but what happened, you said it was a log out but the log shows a log in.

**Comment by DPxKnightxZero** [comment]
Wait what? I don't understand I logged out and It kept crashing after trying to get back in game ... :/ I am going to run a few test I will tell you the results.

**Comment by DPxKnightxZero** [comment]
okay so single player didn't break. I will test it server side now.

**Comment by CannibalVox** [comment]
Crashing when you log back in is very different from crashing when logging out, which is what the title says!

**Comment by DPxKnightxZero** [comment]
okay so it still crashed serverside

Error From Server Console
16.03 10:44:44 [Server] Server thread/INFO [FML]: Unloading dimension 2
16.03 10:44:44 [Server] Server thread/INFO TEMP left the game
16.03 10:44:44 [Disconnect] User TEMP has disconnected, reason: TranslatableComponent{key='disconnect.genericReason', args=[Internal Exception: java.io.IOException: Connection reset by peer], siblings=[], style=Style{hasParent=false, color=null, bold=null, italic=null, underlined=null, obfuscated=null, clickEvent=null, hoverEvent=null}}
16.03 10:44:44 [Server] INFO at java.lang.Thread.run(Thread.java:745) [?:1.8.0_31]
16.03 10:44:44 [Server] INFO at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:101) [SingleThreadEventExecutor$2.class:?]
16.03 10:44:44 [Server] INFO at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:341) [NioEventLoop.class:?]
16.03 10:44:44 [Server] INFO at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:447) [NioEventLoop.class:?]
16.03 10:44:44 [Server] INFO at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:480) [NioEventLoop.class:?]
16.03 10:44:44 [Server] INFO at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:87) [AbstractNioByteChannel$NioByteUnsafe.class:?]
16.03 10:44:44 [Server] INFO at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:208) ~[NioSocketChannel.class:?]
16.03 10:44:44 [Server] INFO at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:870) ~[AbstractByteBuf.class:?]
16.03 10:44:44 [Server] INFO at io.netty.buffer.UnpooledUnsafeDirectByteBuf.setBytes(UnpooledUnsafeDirectByteBuf.java:436) ~[UnpooledUnsafeDirectByteBuf.class:?]
16.03 10:44:44 [Server] INFO at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:379) ~[?:1.8.0_31]
16.03 10:44:44 [Server] INFO at sun.nio.ch.IOUtil.read(IOUtil.java:192) ~[?:1.8.0_31]
16.03 10:44:44 [Server] INFO at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223) ~[?:1.8.0_31]
16.03 10:44:44 [Server] INFO at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39) ~[?:1.8.0_31]
16.03 10:44:44 [Server] INFO at sun.nio.ch.FileDispatcherImpl.read0(Native Method) ~[?:1.8.0_31]
16.03 10:44:44 [Server] INFO java.io.IOException: Connection reset by peer
16.03 10:44:44 [Server] Netty IO #4/ERROR [FML]: NetworkDispatcher exception
16.03 10:44:41 [Multicraft] TEMP ran command Message of the Day
16.03 10:44:41 [Server] Server thread/INFO TEMP joined the game
16.03 10:44:41 [Connect] User TEMP, IP 76.113.166.11
16.03 10:44:41 [Server] Server thread/INFO [FML]: Loading dimension 2 (DeathPlagueGamers) (net.minecraft.server.dedicated.DedicatedServer@265f4aa3)
16.03 10:44:41 [Server] Server thread/INFO [FML]: [Server thread] Server side modded connection established
16.03 10:44:40 [Server] Netty IO #4/INFO [FML]: Attempting connection with missing mods [] at CLIENT
16.03 10:44:40 [Server] Netty IO #4/INFO [FML]: Client attempting to join with 14 mods : HelpFixer@1.0.7,DeathCounter@4.0.0,FML@7.10.85.1291,dimdoors@2.2.5-test,inventorytweaks@1.59-dev-152-cf6e263,FastCraft@1.19,Forge@10.13.2.1291,universalcoins@1.7.10-1.6.5,dungeonpack@1.7.10-1.0,mcp@9.05,BugfixMod@2.5.0,journeymap@5.0.1,zyinhud@1.3.9,AS_MultiMine@1.4.3
16.03 10:44:40 [Server] Netty IO #4/INFO [FML]: Client protocol version 1
16.03 10:44:38 [Server] Server thread/INFO [FML]: Unloading dimension 2
16.03 10:44:37 [Server] Server thread/INFO TEMP left the game
16.03 10:44:37 [Disconnect] User TEMP has disconnected, reason: TextComponent{text='Disconnected', siblings=[], style=Style{hasParent=false, color=null, bold=null, italic=null, underlined=null, obfuscated=null, clickEvent=null, hoverEvent=null}}

ERROR LOG CLIENTSIDE

---- Minecraft Crash Report ----
// On the bright side, I bought you a teddy bear!

Time: 3/16/15 9:43 AM
Description: Ticking screen

java.lang.NullPointerException: Ticking screen
	at StevenDimDoors.mod_pocketDim.core.PocketManager.getDimensionData(PocketManager.java:582)
	at StevenDimDoors.mod_pocketDim.world.PocketProvider.func_76556_a(PocketProvider.java:92)
	at net.minecraft.world.WorldProvider.func_76558_a(WorldProvider.java:45)
	at net.minecraft.world.World.finishSetup(World.java:203)
	at net.minecraft.client.multiplayer.WorldClient.<init>(WorldClient.java:59)
	at net.minecraft.client.network.NetHandlerPlayClient.func_147282_a(NetHandlerPlayClient.java:237)
	at net.minecraft.network.play.server.S01PacketJoinGame.func_148833_a(SourceFile:70)
	at net.minecraft.network.play.server.S01PacketJoinGame.func_148833_a(SourceFile:13)
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212)
	at net.minecraft.client.multiplayer.GuiConnecting.func_73876_c(SourceFile:78)
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1649)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:961)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:887)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

        A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at StevenDimDoors.mod_pocketDim.core.PocketManager.getDimensionData(PocketManager.java:582)
	at StevenDimDoors.mod_pocketDim.world.PocketProvider.func_76556_a(PocketProvider.java:92)
	at net.minecraft.world.WorldProvider.func_76558_a(WorldProvider.java:45)
	at net.minecraft.world.World.finishSetup(World.java:203)
	at net.minecraft.client.multiplayer.WorldClient.<init>(WorldClient.java:59)
	at net.minecraft.client.network.NetHandlerPlayClient.func_147282_a(NetHandlerPlayClient.java:237)
	at net.minecraft.network.play.server.S01PacketJoinGame.func_148833_a(SourceFile:70)
	at net.minecraft.network.play.server.S01PacketJoinGame.func_148833_a(SourceFile:13)
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212)
	at net.minecraft.client.multiplayer.GuiConnecting.func_73876_c(SourceFile:78)

-- Affected screen --
Details:
	Screen name: net.minecraft.client.multiplayer.GuiConnecting
Stacktrace:
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1649)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:961)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:887)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

-- System Details --
Details:
	Minecraft Version: 1.7.10
	Operating System: Windows 8.1 (amd64) version 6.3
	Java Version: 1.8.0_31, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 127309968 bytes (121 MB) / 278945792 bytes (266 MB) up to 4281597952 bytes (4083 MB)
	JVM Flags: 6 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx4G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
	AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	FML: MCP v9.05 FML v7.10.85.1291 Minecraft Forge 10.13.2.1291 Optifine OptiFine_1.7.10_HD_U_B5 14 mods loaded, 14 mods active
	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	FML{7.10.85.1291} [Forge Mod Loader] (forge-1.7.10-10.13.2.1291.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	Forge{10.13.2.1291} [Minecraft Forge] (forge-1.7.10-10.13.2.1291.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	BugfixMod{2.5.0} [BugfixMod] (minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	inventorytweaks{1.59-dev-152-cf6e263} [Inventory Tweaks] ([Client]-InventoryTweaks-1.59-dev-152.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	journeymap{5.0.1} [JourneyMap] ([Client]-JourneyMap5.0.1_Unlimited_MC1.7.10.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	zyinhud{1.3.9} [Zyin's HUD] ([Client]-ZyinsHUD-(1.7.10)-v.1.3.9.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	DeathCounter{4.0.0} [DeathCounter] ([Universal]-DeathCounter-4.0.0.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	dimdoors{2.2.5-test} [Dimensional Doors] ([Universal]-DimensionalDoors-2.2.5-test6.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	dungeonpack{1.7.10-1.0} [Dungeon Pack] ([Universal]-dungeonpack-1.7.10-1.0.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	FastCraft{1.19} [FastCraft] ([Universal]-fastcraft-1.19.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	HelpFixer{1.0.7} [HelpFixer] ([Universal]-HelpFixer-1.0.7.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	AS_MultiMine{1.4.3} [Multi Mine] ([Universal]-MultiMine-1.7.10.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	universalcoins{1.7.10-1.6.5} [Universal Coins] ([Universal]-UniversalCoins-1.7.10-1.6.5.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
	Launched Version: 1.7.10-Forge10.13.2.1291
	LWJGL: 2.9.1
	OpenGL: AMD Radeon HD 7700 Series GL version 4.4.13283 Compatibility Profile Context 14.501.1003.0, ATI Technologies Inc.
	GL Caps: Using GL 1.3 multitexturing.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Anisotropic filtering is supported and maximum anisotropy is 16.
Shaders are available because OpenGL 2.1 is supported.

	Is Modded: Definitely; Client brand changed to 'fml,forge'
	Type: Client (map_client.txt)
	Resource Packs: []
	Current Language: English (US)
	Profiler Position: N/A (disabled)
	Vec3 Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	Anisotropic Filtering: Off (1)

**Comment by CannibalVox** [comment]
If you remove everything but dimdoors and dungeonpack from the client and server does this still happen?

**Comment by DPxKnightxZero** [comment]
Let me check.

**Comment by DPxKnightxZero** [comment]
Yeah it still happens :(

**Comment by CannibalVox** [comment]
Is there any way you can zip up the server and send it to me?  I can't repro this but want to.

**Comment by DPxKnightxZero** [comment]
Yes of course :)

**Comment by DPxKnightxZero** [comment]
If you wanted a beta tester I can do whatever also I have a skype or ts3 if you need extra hands. I have nothing but time. Anyways here is your file link http://bitly.com/DPGDLSF

**Comment by DPxKnightxZero** [comment]
What the heck ... I am  going to re-download the file one sec ...

**Comment by DPxKnightxZero** [comment]
Okay the error is different than when I used the server through a server host.

**Comment by CannibalVox** [comment]
why is that an exe file?  I am extremely nervous about running that.  Please send a zip or rar or 7z or something.

**Comment by DPxKnightxZero** [comment]
I understand ... I usually pack stuff that way I can zip it.

**Comment by DPxKnightxZero** [comment]
Okay I posted a .zip version I didn't use any form of compression just straight zip format :P

**Comment by DPxKnightxZero** [comment]
Okay so whatever inflicts the crash ... it can be rectified after I clear player data for that specific player. I still don't understand what is different between my 24/7 host and The jar on my pc ... oddly enough they are the same >.>

**Comment by DPxKnightxZero** [comment]
Wow I feel really stupid ... I forgot to bind the port he eheheheh :/

**Comment by CannibalVox** [comment]
So wait, did that fix it?

**Comment by DPxKnightxZero** [comment]
Checking now.

**Comment by DPxKnightxZero** [comment]
Okay so nothing I do changes the crash ... 
Test Client 
Steps - Log in game , Create Pocket Dimension, Enter Dimension , Log Out ,  log in, Nothing wrong

Test Dedicated Server
Steps - Log in game , Create Pocket Dimension, Enter Dimension , Log Out , log back in, crash with this error in server log,
[14:23:28] [Netty IO #2/ERROR] [FML]: NetworkDispatcher exception
java.io.IOException: An existing connection was forcibly closed by the remote ho
st
        at sun.nio.ch.SocketDispatcher.read0(Native Method) ~[?:1.8.0_31]
        at sun.nio.ch.SocketDispatcher.read(Unknown Source) ~[?:1.8.0_31]
        at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source) ~[?:1.8.0_31]
        at sun.nio.ch.IOUtil.read(Unknown Source) ~[?:1.8.0_31]
        at sun.nio.ch.SocketChannelImpl.read(Unknown Source) ~[?:1.8.0_31]
        at io.netty.buffer.UnpooledUnsafeDirectByteBuf.setBytes(UnpooledUnsafeDi
rectByteBuf.java:436) ~[UnpooledUnsafeDirectByteBuf.class:?]
        at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:870)
~[AbstractByteBuf.class:?]
        at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketCha
nnel.java:208) ~[NioSocketChannel.class:?]
        at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(Abstra
ctNioByteChannel.java:87) [AbstractNioByteChannel$NioByteUnsafe.class:?]
        at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.jav
a:480) [NioEventLoop.class:?]
        at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEve
ntLoop.java:447) [NioEventLoop.class:?]
        at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:341) [NioEven
tLoop.class:?]
        at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThread
EventExecutor.java:101) [SingleThreadEventExecutor$2.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_31]
[14:23:28] [Server thread/INFO]: TEMP lost connection: TranslatableComponent{key
='disconnect.genericReason', args=[Internal Exception: java.io.IOException: An e
xisting connection was forcibly closed by the remote host], siblings=[], style=S
tyle{hasParent=false, color=null, bold=null, italic=null, underlined=null, obfus
cated=null, clickEvent=null, hoverEvent=null}}
[14:23:28] [Server thread/INFO]: TEMP left the game
[14:23:29] [Server thread/INFO] [FML]: Unloading dimension 2

This error client log
java.lang.NullPointerException: Ticking screen
	at StevenDimDoors.mod_pocketDim.core.PocketManager.getDimensionData(PocketManager.java:582)
	at StevenDimDoors.mod_pocketDim.world.PocketProvider.func_76556_a(PocketProvider.java:92)
	at net.minecraft.world.WorldProvider.func_76558_a(WorldProvider.java:45)
	at net.minecraft.world.World.finishSetup(World.java:203)
	at net.minecraft.client.multiplayer.WorldClient.<init>(WorldClient.java:59)
	at net.minecraft.client.network.NetHandlerPlayClient.func_147282_a(NetHandlerPlayClient.java:237)
	at net.minecraft.network.play.server.S01PacketJoinGame.func_148833_a(SourceFile:70)
	at net.minecraft.network.play.server.S01PacketJoinGame.func_148833_a(SourceFile:13)
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212)
	at net.minecraft.client.multiplayer.GuiConnecting.func_73876_c(SourceFile:78)
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1649)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:961)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:887)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

**Comment by DPxKnightxZero** [comment]
I could be wrong but when logging out of server the reason it breaks is because the server tells the client a player is still logged in and the client is confused because it is clearly flagged the player is not logged in.

**Comment by DPxKnightxZero** [comment]
I think at this point I have provided all the information I can unless you have something you want me to test.

**Comment by DPxKnightxZero** [comment]
Question did you activate any flags when you run your game ... can I have a copy of your game and java version? Lets reverse test this. I will see if I can duplicate issue.

**Comment by keenerb** [comment]
This happens on the test server, canvox.

```
[B#283]  [10:19:03] [Client thread/FATAL]: Reported exception thrown!
[B#283]  net.minecraft.util.ReportedException: Ticking screen
[B#283]  	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1663) ~[bao.class:?]
[B#283]  	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:961) ~[bao.class:?]
[B#283]  	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:887) [bao.class:?]
[B#283]  	at net.minecraft.client.main.Main.main(SourceFile:148) [Main.class:?]
[B#283]  	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_25]
[B#283]  	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_25]
[B#283]  	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_25]
[B#283]  	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_25]
[B#283]  	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) [launchwrapper-1.11.jar:?]
[B#283]  	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.11.jar:?]
[B#283]  Caused by: java.lang.NullPointerException
[B#283]  	at StevenDimDoors.mod_pocketDim.core.PocketManager.getDimensionData(PocketManager.java:582) ~[PocketManager.class:?]
[B#283]  	at StevenDimDoors.mod_pocketDim.world.PocketProvider.func_76556_a(PocketProvider.java:92) ~[PocketProvider.class:?]
[B#283]  	at net.minecraft.world.WorldProvider.func_76558_a(WorldProvider.java:45) ~[aqo.class:?]
[B#283]  	at net.minecraft.world.World.finishSetup(World.java:203) ~[ahb.class:?]
[B#283]  	at net.minecraft.client.multiplayer.WorldClient.<init>(WorldClient.java:59) ~[bjf.class:?]
[B#283]  	at net.minecraft.client.network.NetHandlerPlayClient.func_147282_a(NetHandlerPlayClient.java:237) ~[bjb.class:?]
[B#283]  	at net.minecraft.network.play.server.S01PacketJoinGame.func_148833_a(SourceFile:70) ~[hd.class:?]
[B#283]  	at net.minecraft.network.play.server.S01PacketJoinGame.func_148833_a(SourceFile:13) ~[hd.class:?]
[B#283]  	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212) ~[ej.class:?]
[B#283]  	at net.minecraft.client.multiplayer.GuiConnecting.func_73876_c(SourceFile:78) ~[bcx.class:?]
[B#283]  	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1649) ~[bao.class:?]
[B#283]  	... 9 more
[B#283]  [10:19:03] [Client thread/INFO] [STDOUT]: [net.minecraft.client.Minecraft:func_71377_b:349]: ---- Minecraft Crash Report ----
[B#283]  // Would you like a cupcake?
[B#283]  
[B#283]  Time: 3/17/15 10:19 AM
[B#283]  Description: Ticking screen
[B#283]  
[B#283]  java.lang.NullPointerException: Ticking screen
[B#283]  	at StevenDimDoors.mod_pocketDim.core.PocketManager.getDimensionData(PocketManager.java:582)
[B#283]  	at StevenDimDoors.mod_pocketDim.world.PocketProvider.func_76556_a(PocketProvider.java:92)
[B#283]  	at net.minecraft.world.WorldProvider.func_76558_a(WorldProvider.java:45)
[B#283]  	at net.minecraft.world.World.finishSetup(World.java:203)
[B#283]  	at net.minecraft.client.multiplayer.WorldClient.<init>(WorldClient.java:59)
[B#283]  	at net.minecraft.client.network.NetHandlerPlayClient.func_147282_a(NetHandlerPlayClient.java:237)
[B#283]  	at net.minecraft.network.play.server.S01PacketJoinGame.func_148833_a(SourceFile:70)
[B#283]  	at net.minecraft.network.play.server.S01PacketJoinGame.func_148833_a(SourceFile:13)
[B#283]  	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212)
[B#283]  	at net.minecraft.client.multiplayer.GuiConnecting.func_73876_c(SourceFile:78)
[B#283]  	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1649)
[B#283]  	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:961)
[B#283]  	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:887)
[B#283]  	at net.minecraft.client.main.Main.main(SourceFile:148)
[B#283]  	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[B#283]  	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
[B#283]  	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
[B#283]  	at java.lang.reflect.Method.invoke(Unknown Source)
[B#283]  	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
[B#283]  	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
```

**Comment by CannibalVox** [comment]
Fixed in 91f96c825c22598167cd288a6a76eb8957cc4bcb

Thanks for the report!
